tests/periph/freqm/README.md

```diff
@@ -0,0 +1,14 @@
+Peripheral FREQM Test Application
+=====================================
+
+This application tests the frequency meter (FREQM) functionality:
+
+- measuring the frequency of an internal clock with another internal clock as
+  reference
```

Comment:
What prevent us from measuring an external clock ?

Reply:
> What prevent us from measuring an external clock ?

Nothing, I just wanted to make this test as simple as possible. Do you think a test-application for an external clock should be provided?

Reply:
Not necessarily now. This can be done in a followup PR. But at least, remove the first 'internal' word reference here as it might be confusing.

Reply:
```suggestion
- measuring the frequency of a clock with another internal clock as
  reference
```